src/quick-start.md

```diff
@@ -240,11 +239,28 @@ the Kubewarden stack:
 kubectl delete namespace kubewarden
 ```
 
-> **Note:** kubewarden contains a helm pre-delete hook that will remove all `PolicyServers` and `ClusterAdmissionPolicies`.
+> **Note:** kubewarden contains a helm pre-delete hook that will remove all `PolicyServers` and `kubewarden-controller`.
 > Then the `kubewarden-controller` will delete all resources, so it is important that `kubewarden-controller` is running
-> when helm uninstall is executed. If the controller is not running, and you want to manually remove `PolicyServers` and 
-> `ClusterAdmissionPolicies` resources, you need to remove the kubewarden finalizer. 
+> when helm uninstall is executed. 
 
+Kubewarden pre-delete helm hook will delete all `PolicyServers`. Then the `kubewarden-controller` will do the following for each `PolicyServer`:
+* Delete all `ClusterAdmissionPolices` bounded to it. For each `ClusterAdmissionPolicy` the specified `ValidatingWebhookConfiguration` or `MutatingWebhookConfiguration` will be deleted
+* Once this is done, it will delete the `policy-server` Deployment
+
+`ValidatingWebhookConfigurations` and `MutatingWebhookConfigurations` created by kubewarden should be deleted, this can be checked with:
+
+```shell
+kubectl get validatingwebhookconfigurations.admissionregistration.k8s.io -l "kubewarden" && \
+kubectl get mutatingwebhookconfigurations.admissionregistration.k8s.io -l "kubewarden"
```

Comment:
What is the intent of `-l` here?

Reply:
get all webhooks created by kubewarden (those who have a kubewarden label)

Reply:
I completely missed we were creating webhooks with `kubewarden: "true"`...